test/python/evaluation/test_tool_path.py

```diff
@@ -0,0 +1,24 @@
+from test.python.evaluation import XLSX_DATA_FOLDER
+from test.python.evaluation.testing_config import get_testing_arguments
+
+import pandas as pd
+import pytest
+from src.python import MAIN_FOLDER
+from src.python.evaluation.evaluation_config import EvaluationConfig
+from src.python.evaluation.xlsx_run_tool import create_dataframe
+
+
+def test_correct_tool_path():
+    testing_arguments_dict = get_testing_arguments(n_args=5)
+    testing_arguments_dict['xlsx_file_path'] = XLSX_DATA_FOLDER / 'test_unsorted_order.xlsx'
+    config = EvaluationConfig(testing_arguments_dict)
+    assert type(create_dataframe(config)) == pd.DataFrame
```

Comment:
Please, don't use `Union[int, pd.DataFrame]` and assert like this. Assert content or something like this or assert raise error, don't type